gap/precompiled_categories/CategoryOfSkeletalFinSetsPrecompiled.gi

```diff
@@ -304,6 +304,27 @@ end
         
     , 100 );
     
+    ##
+    AddExactCoverWithGlobalElements( cat,
+        
+########
+function ( cat_1, arg2_1 )
+    local hoisted_1_1, hoisted_2_1, hoisted_3_1, deduped_4_1, deduped_5_1;
+    deduped_5_1 := Length( arg2_1 );
+    deduped_4_1 := [ 0 .. deduped_5_1 - 1 ];
+    hoisted_3_1 := CreateCapCategoryObjectWithAttributes( cat_1, Length, 1 );
+    hoisted_2_1 := deduped_5_1;
+    hoisted_1_1 := deduped_4_1;
+    return List( deduped_4_1, function ( i_2 )
+            local deduped_1_2;
+            deduped_1_2 := CAP_JIT_INCOMPLETE_LOGIC( hoisted_1_1[1 + i_2] );
```

Comment:
For the already existing logic template

```
CapJitAddLogicTemplate(
    rec(
        variable_names := [ "pos_end", "index" ],
        variable_filters := [ IsInt, IsInt ],
        src_template := "[ 0 .. pos_end ][index]",
        dst_template := "index - 1",
    )
);
```

it is probably too late. Right?

Reply:
Correct, this occurs all over the place.